sa/database.go

```diff
@@ -156,12 +160,103 @@ func NewDbMapFromConfig(config *mysql.Config, settings DbSettings) (*boulderDB.W
 	dbmap := &gorp.DbMap{Db: db, Dialect: dialect, TypeConverter: BoulderTypeConverter{}}
 
 	initTables(dbmap)
-
 	return &boulderDB.WrappedMap{DbMap: dbmap}, nil
 }
 
-// adjustMySQLConfig sets certain flags that we want on every connection.
-func adjustMySQLConfig(conf *mysql.Config) {
+var checkQuoteRE = regexp.MustCompile(`^'{1}\w+'{1}$`)
+var checkImproperIntRE = regexp.MustCompile(`^'{1}\d+'{1}$`)
```

Comment:
I think that `x{1}` is redundant in a regex: it's equivalent to `x`. The curly brace syntax is only useful for things like `x{2-5}`, as a point in between just `x` and the infinite-repetition of `x+` or `x*`.